### ISSUE TITLE
text not changed after calling function

### ISSUE THREAD

**Issue by gerard0315** [eyaleizenberg/react-native-floating-label-text-input#13]
Hi, tried the package, looks awesome, but the input text is not changed after modifications

**Comment by eyaleizenberg** [comment]
Hi,
Can you please give more information? steps to recreate? Version? etc..

**Comment by gerard0315** [comment]
I am using RN 0.24.1, this is my component
    <FloatLabelTextInput
        selectionColor = "#D25061"
        placeholder={"Name Here"}
        value={this.state.Name}
        onChangeTextValue={this.onNameInput}
        />

this is onNameInput function:

```
    onNameInput: function(event) {
        console.log('editing Name');
        this.setState({ Name: event.nativeEvent.text });
        console.log(this.state.Name);
},
```

tried to change `onChangeText={this.setText}` to `onChange = {this.setText}`, it actually worked but gave a type warning

**Comment by eyaleizenberg** [comment]
But it looks like you are setting the state to dogName and not Name

**Comment by gerard0315** [comment]
yeah sorry thats a typo in the post but not in the actual code, fixed that.

**Comment by eyaleizenberg** [comment]
Is this still an issue?
